modules/fedimint-lnv2-tests/src/bin/tests.rs

```diff
@@ -16,10 +16,6 @@ use tracing::info;
 #[derive(Parser)]
 enum TestOpts {
     All,
-    GatewayRegistration,
-    SelfPaymentsRefund,
-    SelfPaymentsSuccess,
-    LightningPayment,
     InterFederation,
```

Comment:
Looks like both on master and in this PR, `TestOpts::All` is running all tests except for inter-federation tests. Is there a reason for that? If so, can you document it?

Reply:
The remaining two variants are tests that are run in CI in parallel, the other one are not used for that.